log4j-api/src/main/java/org/apache/logging/log4j/message/ParameterizedMessage.java

```diff
@@ -302,6 +302,10 @@ public static String deepToString(final Object o) {
         return ParameterFormatter.deepToString(o);
     }
 
+    public static void deepToString(final Object o, final StringBuilder buf) {
```

Comment:
I think this would make more sense in the StringBuilders utility class, but I put it here for uniformity with the overload.

Reply:
Fine by me. That said, @remkop, @jvz, and/or @rgoers might have a different opinion on this.